src/components/Calender/CalendarSelector.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { Calendar } from './Calendar';
+import { Icon, Layout, Text } from '@ui-kitten/components';
+
+import { View, StyleSheet } from 'react-native';
+import { months } from '../../services/Calendar/models';
+import { useStringDate } from '../../services/Hooks/useStringDate';
+import { selectSwapDate } from '../../store/actions';
+import { useDispatch, useSelector } from 'react-redux';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+export const CalendarSelector = (props) => {
+    const [displayedDate, setDisplayedDate] = useState(props.displayedDate);
```

Comment:
since with props being `props.displayedDate`, you could make hook names currentDate (?)